### PR TITLE
one-liner for a prod-compatible service

### PR DESCRIPTION
**Pull request by benjigoldberg**
@briancharous what do you think of this approach? I think we define one for http, and someday one for grpc. A user could take the two cobra commands and chain them together, as well.

### CODE REVIEW

File: service/http.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		Fields: map[string]interface{}{
+			"version": hc.Version,
+			"git_sha": hc.GitSHA,
+		},
+		Cores: []zapcore.Core{&tools.SentryCore{}},
+	}
+	cmd := &cobra.Command{
+		Use:              hc.Name,
+		Short:            "Starts and runs an HTTP Server",
+		Long:             `Starts and runs an HTTP Server`,
```
Comment (briancharous) [inline service/http.go]: Change to standard quotes

File: service/http.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		Version:          fmt.Sprintf("%s (%s)", hc.Version, hc.GitSHA),
+		PersistentPreRun: tools.CobraBindEnvironmentVariables(hc.Name),
+		RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (briancharous) [inline service/http.go]: If this can't return an error just use `Run` instead of `RunE`

**Comment by benjigoldberg** [comment]
also codecov is full of it, coverage on kafka did not go down lol

**Pull request merged**